DEPLOY.rst

```diff
@@ -6,57 +6,128 @@
     under the terms of the MIT License; see LICENSE file for more details.
 
 
-Deploying
-=========
+Deploy
+======
 
+This section explains how to get the ``Cube-Builder-AWS`` up and running on `Amazon Web Services <https://aws.amazon.com/>`_.
+If you do not read yet the :doc:`installation`, take a look at this tutorial on how to install it in your system in devmode
+and be familiar with Python module.
 
-Create infrastructure
----------------------
 
-.. code-block:: shell
+.. warning::
+
+    Make sure to identify which region the dataset is available.
+    For example, most of `GEO Earth datasets <https://aws.amazon.com/earth/>`_ like ``Sentinel-2``, ``Landsat-8`` are 
+    stored in ``Oregon`` (``us-west-2``). In this tutorial, we are going to use ``us-west-2``.
+
+    If you generate any data cube that are in a different region of BDC services, you may face high cost charges in the billing.
+
+
+
+.. requirements:
+
+Requirements
+------------
+
+- `RDS PostgreSQL <https://aws.amazon.com/rds/postgresql/>`_: A minimal instance of PostgreSQL database with PostGIS support.
+  The ``instance_type`` depends essentially how many parallel processing ``Lambdas`` are running. For this example,
```

Comment:
The ``instance_type`` depends essentially on how many parallel processing ``Lambdas`` are running.